RawDataUnpacker/SSDUnpacker.h

```diff
@@ -0,0 +1,29 @@
+////////////////////////////////////////////////////////////////////////
+/// \brief   Unpack class with static functions to convert SSD data 
+///          slightly more useful raw data products
+/// \author  jpaley@fnal.gov
+////////////////////////////////////////////////////////////////////////
+#ifndef SSDRAWDATA_UNPACK
+#define SSDRAWDATA_UNPACK
+
+#include <vector>
+#include <iostream>
+#include <utility>
+#include <fstream>
```

Comment:
i prefer these alphabetized, and separate by newline the art/emph includes...again just pedantic